src/main/java/graphql/execution/SubscriptionExecutionStrategy.java

```diff
@@ -92,15 +92,16 @@ public SubscriptionExecutionStrategy(DataFetcherExceptionHandler dataFetcherExce
      */
 
     private CompletableFuture<ExecutionResult> executeSubscriptionEvent(ExecutionContext executionContext, ExecutionStrategyParameters parameters, Object eventPayload) {
-        ExecutionContext newExecutionContext = executionContext.transform(builder -> builder.root(eventPayload));
+        ExecutionContext newExecutionContext = executionContext.transform(builder -> builder
+                .root(eventPayload)
+                .resetErrors()
+        );
```

Comment:
We reset errors so that we have errors per streamed event BUT we use the same other execution context